cli_tools_e2e_test/gce_image_import_export/test_suites/scripts/post_translate_test.ps1

```diff
@@ -0,0 +1,118 @@
+#  Copyright 2017 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+$ErrorActionPreference = 'Stop'
+
+function Check-VMWareTools {
```

Comment:
May I know how you determine which things to validate here?

Reply:
I referenced the current windows 2019 script for daisy integration tests. I created a new file here because the instance signals are changed to the same as the signals in the ubuntu test script. The way, we can reuse the workflow file since its the same workflow for both ubuntu and windows instance.